eli5/keras/gradcam.py

```diff
@@ -90,58 +36,66 @@ def gradcam_backend(model, # type: Model
     targets : list, optional
         Index into the network's output,
         indicating the output node that will be
-        used as the "loss" during differentiation.
+        used as the "loss" or scalar target during differentiation.
 
     activation_layer : keras.layers.Layer
         Keras layer instance to differentiate with respect to.
-    
 
-    See :func:`eli5.keras.explain_prediction` for description of the 
-    ``model``, ``doc``, ``targets`` parameters.
+
+    See :func:`eli5.keras.explain_prediction` for description of the
+    ``model``, ``doc``, and ``targets`` parameters.
 
     Returns
     -------
-    (weights, activations, gradients, predicted_idx, predicted_val) : (numpy.ndarray, ..., int, float)
+    (activations, gradients, predicted_idx, predicted_val) :
+        (numpy.ndarray, numpy.ndarray, int, float)
```

Comment:
Last two arguments of returned tuple have different types in the type annotation (np.ndarrays) and in the docstring (int, float) - would be great to sync them if possible.

Reply:
Good spot. Forgot to update the docstring